examples/qt/README.md

```diff
@@ -0,0 +1,45 @@
+# libsigc++ with Qt
+
+It is possible to use libsigc++ with Qt.  However, because of the signals/slots
+mechanism of Qt, there is some setup that must be done in order for this to
+happen correctly.
+
+Steps to use libsigc++ with Qt:
+1. In your .pro file, add `CONFIG += no_keywords`.  This configures Qt to not
+define the keywords `emit`, `signals`, and `slot`.
```

Comment:
These are macros, not keywords.

Reply:
They're macros in the C++ land, but that is paraphrasing what the Qt documentation says in the link you put above: 

> It tells Qt not to define the moc keywords `signals`, `slots`, and `emit` [...]